style.css

```diff
@@ -541,6 +679,74 @@ nav a {
     margin-top: 28px;
   }
 
+  /************* pop-up window *****************/
+  .pop-up {
+    width: 716px;
+  }
+
+  #pop.active {
+    top: 2%;
+  }
+
+  .main-text {
+    margin-bottom: 15px;
+  }
+
+  .project-image {
+    width: 470px;
+    height: 300px;
+  }
```

Comment:
- [ ] Kindly consider giving your image a width and height of `100%` so it spans across the entire width of its container. This way your image will look exactly like that of the `Figma` design and look great to create a good user experience. See below for more details: 
<table>
<tr>
<th align="middle">Figma design</th>
<th align="middle">Your design</th>
</tr>
<tr>
<td>
<img width="721" alt="Screenshot 2023-05-19 at 10 24 06" src="https://github.com/Fombi-Favour/Microverse-portfolio/assets/107192076/1555d4ba-5cd1-4688-8a0a-97a6884fd4f1">
</td>
<td>
<img width="1041" alt="Screenshot 2023-05-19 at 10 27 36" src="https://github.com/Fombi-Favour/Microverse-portfolio/assets/107192076/86822e88-db4f-45d7-8ae9-f8d43d6b94ed">
</td>
</tr>
</table>